pkg/doc/presexch/definition.go

```diff
@@ -298,8 +298,14 @@ func makeRequirement(requirements []*SubmissionRequirement, descriptors []*Input
 	return req, nil
 }
 
+// CredentialCandidate contains the necessary information for processing the credential.
+type CredentialCandidate struct {
+	Opts       []verifiable.CredentialOpt
```

Comment:
I don't think we need to specify options for every credential. Can you explain why we do?

All `verifiable.CredentialOpts` should apply equally to all credentials. For example the JSONLD document loader configured in the unit test.

Reply:
I would probably refactor as: `CreateVP(credentials []*verifiable.Credential, options ...verifiable.CredentialOpt)`

Reply:
ok, I will rollback it. I did it bc I don't know what else might be needed :) working with structure is easier comparing to function arguments

Reply:
> comparing to function arguments

Ehm - to be clear, I was showing an example of how I would refactor the existing `CreateVP` function in my comment above. I was not suggesting functional arguments for `CreateVP`

Reply:
Done!